code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -73,9 +97,6 @@
 /turf/open/floor/plating/asteroid/basalt/Initialize()
 	. = ..()
 	set_basalt_light(src)
-	GET_COMPONENT(arch, /datum/component/archaeology)
-	ASSERT(isnull(arch.callback))
-	arch.callback = CALLBACK(src, /atom/proc/set_light, 0)
```

Comment:
Is this call being maintained? It's used to make glowing turfs stop glowing when they're dug.

Reply:
it wasn't before, now is.